### PR TITLE
Supernode support seed task pr

### PR DESCRIPTION
**Pull request by hhhhsdxxxx**

### CODE REVIEW

**Comment by lowzj** [comment]
There are 6 commits in this pull request. Could you submit several pull requests to separate them?

**Comment by hhhhsdxxxx** [comment]
> There are 6 commits in this pull request. Could you submit several pull requests to separate them?

First PR is ready now

File: apis/swagger.yml [hunk 1]
```diff
@@ -1511,6 +1511,27 @@ definitions:
           This dfget is treated a client and carries a client ID.
           Thus, multiple dfget processes on the same peer have different CIDs.
 
+  HeartBeatResponse:
```
Comment (lowzj) [inline apis/swagger.yml:1514]: Where is the `HeartBeatResponse` be used in swagger?

Comment (hhhhsdxxxx) [reply apis/swagger.yml:1514]: API /peer/heartbeat.

Comment (lowzj) [reply apis/swagger.yml:1514]: Yeah, I find this api, but its response definition is not `HeartBeatResponse`.

Comment (hhhhsdxxxx) [reply apis/swagger.yml:1514]: The definition in file api.md?

Comment (hhhhsdxxxx) [reply apis/swagger.yml:1514]: If so, i'll make it complete soon.

File: supernode/server/router.go [hunk 2]
```diff
@@ -48,6 +48,8 @@ func initRoute(s *Server) *mux.Router {
 		{Method: http.MethodGet, Path: "/peer/piece/suc", HandlerFunc: s.reportPiece},
 		{Method: http.MethodGet, Path: "/peer/service/down", HandlerFunc: s.reportServiceDown},
 		{Method: http.MethodGet, Path: "/peer/piece/error", HandlerFunc: s.reportPieceError},
+		{Method: http.MethodPost, Path: "/peer/network", HandlerFunc: s.getP2PNetworkInfo},
```
Comment (lowzj) [inline supernode/server/router.go:51]: Why is this POST method named `getP2PNetworkInfo`? Is it should be `GET` method?

Comment (hhhhsdxxxx) [reply supernode/server/router.go]: fix it

**Comment by hhhhsdxxxx** [comment]
@lowzj Are there any more comments?

**Pull request merged**